spec/gorouter_templates_spec.rb

```diff
@@ -344,25 +343,24 @@
         end
       end
 
-      describe 'max_(request_)header_kb' do
+      describe 'max_request_header_kb' do
         context 'as a default' do
           it 'should set max_request_header_bytes' do
             expect(parsed_yaml['max_request_header_bytes']).to eq(1_048_576)
           end
         end
 
-        context 'when only max_header_kb is set' do
+        context 'when only max_request_header_kb is set' do
           before do
-            deployment_manifest_fragment['router']['max_header_kb'] = 10
+            deployment_manifest_fragment['router']['max_request_header_kb'] = 10
           end
           it 'should set max_request_header_bytes according to it' do
             expect(parsed_yaml['max_request_header_bytes']).to eq(10_240)
           end
         end
 
-        context 'when max_header_kb and max_request_header_kb are set' do
+        context 'when max_request_header_kb and max_request_header_kb are set' do
```

Comment:
you can delete this test since it doesn't make sense in this new world without max_header_kb